docs/developer/payments/overview.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Payments Overview
+sidebar_label: Overview
+---
+
+There are few ways to work with payments in Saleor:
+
+### Payment apps
+
+Payment apps allow to process payments with external payment providers, without communicating with them directly, which is specially convenient for front-end development.
+
+- Using [payment app](/developer/payments/payment-apps.mdx).
+
+- Guide to [building payment apps](/developer/extending/apps/building-payment-app.mdx).
```

Comment:
IMHO, the second link is a "part" of `Using [payment app]`. As a result of the guide you will get the payment app which uses the API explained in `(/developer/payments/payment-apps.mdx).`
Second option of building an app is making it fully custom. By saying fully custom I mean a case (only as a example), where you host own checkout flow, and provide Saleor only a results. In that case you will use mutations like: `transactionCreate`, `transactionUpdate` and `orderCreateFromCheckout`.
Ref: https://docs.saleor.io/developer/payments#what-is-a-custom-app

Reply:
I have a bit of problem with definition of Custom App vs Payment App, imo these are the same things is just one is using some webhooks and the other are not, but in reality this can be a single app that does both things. I think we need to rethink the overall grouping of terms. Probably the guide should be split between using payments vs building payments